addons/im_livechat/tests/test_message.py

```diff
@@ -72,6 +72,10 @@ def test_message_format(self):
             'recipients': [],
             'record_name': "test1 Ernest Employee",
             'res_id': channel_livechat_1.id,
+            'sender': {
+                'id': self.users[0].partner_id.id,
+                'name': self.users[0].partner_id.name,
```

Comment:
`id` is ready from model because we can't "guess" it, but the name should be determined by the setup, not be dynamic